dev/TeachingTip/TeachingTip.cpp

```diff
@@ -13,16 +13,32 @@ TeachingTip::TeachingTip()
     __RP_Marker_ClassById(RuntimeProfiler::ProfId_TeachingTip);
     SetDefaultStyleKey(this);
     EnsureProperties();
+    m_automationNameChangedCallback = RegisterPropertyChangedCallback(winrt::AutomationProperties::NameProperty(), { this, &TeachingTip::OnAutomationNameChanged });
+    m_automationIdChangedCallback = RegisterPropertyChangedCallback(winrt::AutomationProperties::AutomationIdProperty(), { this, &TeachingTip::OnAutomationIdChanged });
```

Comment:
There's an auto_revoke helper version of this called RegisterPropertyChanged (from CppWinRTHelpers.h). Prefer that.